content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,207 @@
+use to create an event track.
+
+.. image:: track_manage_talks/event-track-form.png
+   :align: center
+   :alt: Typical event track form in the Odoo Events application.
+
+Start by giving this track a :guilabel:`Title`.
+
+Then, if desired, add an accompanying image to the track via the :icon:`fa-pencil`
+:guilabel:`(pencil)` icon that appears when the cursor hovers over the :icon:`fa-camera`
+:guilabel:`(camera)` icon in the upper-right corner of the form. When clicked, proceed to upload the
+desired image for the track.
+
+Next, apply a :guilabel:`Track Date` for the track, and designate a :guilabel:`Location` where the
+talk is planning to be held.
+
+Then, add a :guilabel:`Duration` to the track.
+
+If the :guilabel:`Live Broadcast` setting has been enabled, users have the option to add a
+corresponding link in the :guilabel:`YouTube Video Link` field.
+
+If the :guilabel:`Always Wishlisted` checkbox is ticked, the talk is automatically set as a favorite
+for each registered attendee.
+
+Assign an internal user to be in charge of managing this event in the :guilabel:`Responsible` field.
+By default, the user who initially created the track is assigned.
+
+Then, ensure the track is applied to the correct event in the :guilabel:`Event` field. By default,
+this field is populated with the event related to the *Event Tracks* page the track was originally
+created from.
+
+Next, choose to add existing tags, or create new ones, to further organize the track. These tags,
+and their corresponding tag categories are utilized on the event specific website - mainly on the
+*Talks* web page on the event website, via the drop-down filter menus.
+
+Beneath that general information section, there are three tabs: :guilabel:`Speaker`,
+:guilabel:`Description`, and :guilabel:`Interactivity`.
+
+Speaker tab
+~~~~~~~~~~~
+
+The :guilabel:`Speaker` tab on an event track form is filled with various fields related to the
+specific speaker who is planning to conduct/host the track.
+
+.. image:: track_manage_talks/speaker-tab.png
+   :align: center
+   :alt: The Speaker tab on an event track form in the Odoo Events application.
+
+Contact Details section
+***********************
+
+In the :guilabel:`Contact Details` section, proceed to use the :guilabel:`Contact` drop-down field
+to select an existing contact from the database as the main point of contact for the talk.
+
+If this contact is not yet in the database, type in the name of the contact, and click
+:guilabel:`Create` to create and edit the contact form later, or click :guilabel:`Create and
+edit...` to be taken to that new contact's contact form, where the rest of their pertinent
+information can be entered.
+
+The :guilabel:`Contact Email` and :guilabel:`Contact Phone` fields are greyed-out and populated with
+the information found on that chosen contact's contact form. These fields are not modifiable.
+
+Speaker Bio section
```

Comment:
```suggestion
Speaker Bio section
```

This section was pre-populated based on the contact I selected in the Contact Details section

Reply:
What is this section used for? To show on the website or for the organizer to introduce the speaker?